specification/components/examples/Immunization/POST-mockFLUcode1-Immunization.json

```diff
@@ -0,0 +1,222 @@
+      "identifier": [
+        {
+          "system": "https://fhir.hl7.org.uk/Id/nmc-number",
+          "value": "99A9999A"
+        }
+      ]
+    },
+    {
+      "resourceType": "Patient",
+      "id": "Pat1",
+      "identifier": [
+        {
+          "extension": [
+            {
+              "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-NHSNumberVerificationStatus",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-NHSNumberVerificationStatusEngland",
+                    "code": "01",
+                    "display": "Number present and verified"
+                  }
+                ]
+              }
+            }
+          ],
+          "system": "https://fhir.nhs.uk/Id/nhs-number",
+          "value": "9000000009"
+        }
+      ],
+      "name": [
+        {
+          "family": "Taylor",
+          "given": ["Sarah"]
+        }
+      ],
+      "gender": "unknown",
+      "birthDate": "1965-02-28",
+      "address": [
+        {
+          "postalCode": "EC1A 1BB"
+        }
+      ]
+    },
+    {
+      "resourceType": "QuestionnaireResponse",
+      "id": "QR1",
+      "status": "completed",
+      "item": [
+        {
+          "linkId": "Immunisation",
+          "answer": [
+            {
+              "valueReference": {
+                "reference": "#"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "Consent",
+          "answer": [
+            {
+              "valueCoding": {
+                "code": "310375005",
+                "system": "http://snomed.info/sct",
+                "display": "Immunization consent given (finding)"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "CareSetting",
+          "answer": [
+            {
+              "valueCoding": {
+                "code": "413294000",
+                "system": "http://snomed.info/sct",
+                "display": "Community health services (qualifier value)"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "ReduceValidation",
+          "answer": [
+            {
+              "valueBoolean": false
+            }
+          ]
+        },
+        {
+          "linkId": "LocalPatient",
+          "answer": [
+            {
+              "valueReference": {
+                "identifier": {
+                  "system": "https://ACME/identifiers/patient",
+                  "value": "ACME-patient123456"
+                }
+              }
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "extension": [
+    {
+      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-VaccinationProcedure",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "system": "http://snomed.info/sct",
+            "code": "822851000000102",
```

Comment:
Inconsistent file naming. Can this be changed so that either all these json files are named after the vaccine type, or all are named after the specific VaccinationProcedure code used (as opposed to a mixture of the two)?

Reply:
done.

Reply:
It's the flu file that needs changing (i.e. call the file "POST-822851000000102" to match the vaccinationProcedure code

Reply:
![image](https://github.com/NHSDigital/immunisation-fhir-api/assets/35727498/acd718b3-7e63-445d-9746-f492424c4008)

Reply:
I've changed it to match the contents and changed it in the code.